zulipterminal/ui_tools/boxes.py

```diff
@@ -781,7 +781,7 @@ def transform_content(self) -> Tuple[None, Any]:
         soup = BeautifulSoup(self.message['content'], 'lxml')
         body = soup.find(name='body')
         if body and body.find(name='blockquote'):
-            padding_char = '░'
+            padding_char = QUOTED_TEXT_PADDING
             self.indent_quoted_content(soup, padding_char)
```

Comment:
Maybe combine these two lines now, as with the other case?

I'm aware the name likely came from the variable, but maybe `QUOTED_TEXT_MARKER`?